### DISCUSSION TITLE
What is stopping from supporting Debian officially?

### DISCUSSION THREAD

**Discussion by 0oAstro** [pacstall/pacstall#99] [Q&A] [answered]
Greetings,
I always wonder why pacstall is officially for Ubuntu even though Ubuntu is based on Debian so many more distros, and we can just make pacstall AUR for Debian or DebianUserRepo (DUR). Sure it requires to analyse code and testing but that will payoff as Debian is Mack daddy of distros and we will be supporting literally every distro (excluding Arch based and RHEL&Fedora based distros).

**Comment by Elsie19** [comment]
The problem is package names. Another is that to support multiple distro's would mean that **every** package would need to include both dependencies

**Comment by Elsie19** [comment, marked as answer]
Coming back to this. I’m running Debian with Pacstall and everything is working smoothly. I would say Pacstall works with Debian, but take everything with a grain of salt.

Edit (19/10/22): Still no issues
Edit (24/12/22): Still no issues
Edit (14/11/25): Still no issues :)

**Reply by D-Brox** [reply]
Most debian packages are name compatible with ubuntu

**Comment by trymeouteh** [comment]
Would like to see official support for Debian. I use Ubuntu based distros for PPAs and the Ubuntu repository, but if Pacstall can work on Debian and once Pacstall has many packages like the AUR, no need to rely on Ubuntu anymore.

**Reply by wizard-28** [reply]
We already kind of have official support for Debian. It's just that if a package **only** works on Ubuntu we prevent it's installation on Debian.

**Reply by trymeouteh** [reply]
I assumed the support was unofficial since on the homepage of the pacstall.dev site it says "The AUR for Ubuntu". I would advise that this gets changed to "The AUR for Debian" or "The AUR for Debian and Ubuntu" for newcomers to know that pacstall works on Debian and Ubuntu.
